prime-router/src/main/kotlin/JurisdictionalFilters.kt

```diff
@@ -49,7 +49,11 @@ class Matches : JurisdictionalFilter {
         if (args.size != 2) error("Expecting two args to filter $name:  (columnName, regex)")
         val columnName = args[0]
         val pattern = args[1]
-        return table.stringColumn(columnName).matchesRegex(pattern)
+        val columnNames = table.columnNames()
+        return if (columnNames.contains(columnName))
+            table.stringColumn(columnName).matchesRegex(pattern)
+        else
+            Selection.withRange(0, 0)
     }
```

Comment:
This code would throw an execption if the `columnName` didn't exist in the input schema. Now it returns an empty selection.